cypress/fixtures/listingDetails.json

```diff
@@ -0,0 +1,239 @@
+{
+	"test": true,
+	"listing": {
+		"unitSummaries": {
+			"reserved": null,
+			"general": [
+				{
+					"unitType": "1 BR",
+					"totalUnits": 3,
+					"minSquareFt": 1100,
+					"minRentalMinIncome": 0,
+					"minPriceWithParking": null,
+					"minPriceWithoutParking": null,
+					"minPercentIncome": null,
+					"minOccupancy": 1,
+					"minMonthlyRent": 750,
+					"minHoaDuesWithParking": null,
+					"minHoaDuesWithoutParking": null,
+					"maxSquareFt": 1100,
+					"maxRentalMinIncome": 0,
+					"maxPriceWithParking": null,
+					"maxPriceWithoutParking": null,
+					"maxPercentIncome": null,
+					"maxOccupancy": 3,
+					"maxMonthlyRent": 750,
+					"maxHoaDuesWithParking": null,
+					"maxHoaDuesWithoutParking": null,
+					"listingID": "a0W8H000000AmpKUAS",
+					"availability": 3,
+					"absoluteMinIncome": null,
+					"absoluteMaxIncome": 11529
+				}
+			]
+		},
+		"reservedDescriptor": null,
+		"prioritiesDescriptor": null,
+		"listingID": "a0W8H000000AmpKUAS",
+		"chartTypes": [
+			{
+				"year": "2020",
+				"percent": 120,
+				"numOfHousehold": null,
+				"chartType": "MOHCD",
+				"amount": null
+			}
+		],
+		"attributes": {
+			"type": "Listing",
+			"url": "/services/data/v35.0/sobjects/Listing/a0W8H000000AmpKUAS"
+		},
+		"Name": "Test - Closed with Lottery Results (Do Not Modify)",
+		"Realtor_Commission_Unit": "percent",
+		"Allows_Realtor_Commission": false,
+		"Parking_Information": "Onsite parking",
+		"Tenure": "New rental",
```

Comment:
suggestion (non-blocking): update the name of this file to indicate that it has details for a _rental_ listing.

Reply:
for sure! I also included that it is a completed listing